lib/v2/datasource/remote/model/transaction_model.dart

```diff
@@ -38,19 +43,41 @@ class TransactionModel extends Equatable {
       to: json['act']['data']['to'],
       quantity: json['act']['data']['quantity'],
       memo: json['act']['data']['memo'],
-      timestamp: json['block_time'],
+      timestamp: parseTimestamp(json['block_time']),
       transactionId: json['trx_id'],
       //json["block_num"], // can add this later - neat but changes cache structure
     );
   }
 
+  static TransactionModel? fromTransaction(GenericTransactionModel genericModel) {
```

Comment:
is this method used anywhere?

Reply:
Yes. Very important.

Reply:
this is how we can tell whether or not a transaction is a transfer and we can create the transfer model from the generic one.
Used in the other PRs.